pkg/runtime/cache/namespace.go

```diff
@@ -72,40 +76,45 @@ func NewNamespaceCache(clientset kubernetes.Interface, log logr.Logger) *Namespa
 	return &NamespaceCache{
 		informer:       sharedInformer,
 		log:            log.WithName("cache.namespace"),
+		listenNamespace: listenNamespace,
 		namespaceInfos: make(map[string]*namespaceInfo),
 	}
 }
 
-// isIgnoredNamespace returns true if an object is of type corev1.Namespace
-// and it metadata name is one of 'ack-system', 'kube-system' or 'kube-public'
-func isIgnoredNamespace(raw interface{}) bool {
+// Check if the provided namespace should be listened to or not 
+func isListenNamespace(raw interface{}, listenNamespace string) bool {
 	object, ok := raw.(*corev1.Namespace)
-	return ok &&
-		(object.ObjectMeta.Name == "ack-system" ||
-			object.ObjectMeta.Name == "kube-system" ||
-			object.ObjectMeta.Name == "kube-public")
+	if !ok {
+		return false
+	} 
+
+	if listenNamespace == "" {
+		return true
```

Comment:
I think it's still valid to check that the namespace is not in `kube-system` or `kube-public` (`ack-system` is probably fine?).

Reply:
done

Reply:
IMO we should keep ignoring `ack-system`... But i'm still hesitant about this - what if a user installs a controller in a different namespace than `ack-system`?